app/views/home/faqs.html.erb

```diff
@@ -0,0 +1,40 @@
+<main class="container-fluid">
+  <section class="faqs">
+    <div class="faqs-text">
+      <h3>What is Let Me Help You?</h3>
+      <p>
+      Terapify is a safe and affordable online psychological therapy platform. <br>
+      Our mission is to help people regain the emotional stability they need for their daily lives.
+      </p>
+    </div>
+    <div class="faqs-text">
+      <h3>How does it work?</h3>
+      <p>
+      In Terapify you can speak safely and privately with a trusted psychologist by video call, call or chat. <br>
+      To have an online consultation with one of our psychologists, you have to make an appointment in advance. <br>
+      The availability of hours varies according to the agenda of the psychologist you select. 
+      </p>
+    </div>
+    <div class="faqs-text">
+      <h3>Are online consultations effective? </h3>
+      <p>
+      Online consultations allow you to receive the same psychological care that you would receive in a traditional office, but remotely. <br>
+      You can talk with your therapist by chat without leaving home. 
+      </p>
+    </div>
+    <div class="faqs-text">
+      <h3>When is the appointment payment made?</h3>
+      <p>
+      At the time of scheduling your appointment you must make the payment for it so that it is confirmed with your psychologist and you can carry it out.
+      </p>
+    </div>
+    <div class="faqs-text">
+      <h3>How long is an appointment?</h3>
+      <p>
+      Appointments last 50 minutes. <br>
+      It is important that you enter your appointment on time. The consultation times have been programmed especially for you, if you are late for your appointment, it will have to be shorter to respect the schedules of the other patients. <br>
+      Regardless of the duration of your appointment, it will be charged in full. 
+      </p>
+    </div>
+  </section>
+</main>
```

Comment:
I suggest you use internationalization, so you can add the FAQ in multiple languages.

[Rails Internationalization (I18n)](https://guides.rubyonrails.org/i18n.html)